frontend/src/components/quiz/Questions.js

```diff
@@ -40,7 +40,7 @@ const Questions = () => {
 
     const shuffleAnswers = (event) => {
         event.preventDefault();
-        if (event.target.id="start"){
+        if (event.target.id === "start"){
```

Comment:
Ah the good ole assignment instead of equality mistake, we've all been there XD

Reply:
To be fair though, if this function is attached to only one button, you won't need to check for the event target id, because there is only ever one button that will trigger it

Reply:
theres 2 that trigger it, one that opens the 'curtains' and shuffles the answer & the submit answer

Reply:
there was also a larger problem with the curtains where they dragged the results off screen, which was the main issue!